certified-connectors/Telnyx SMS/readme.md

```diff
@@ -0,0 +1,18 @@
+# Telnyx
+Telnyx Suite is a comprehensive cloud communication platform that provides communication capabilities via voice, SMS, fax, and IP services. Telnyx provides global high-speed connections for low latency, high uptime service. The Telnyx Suite includes SIP trunking, messaging API, voice API, lookup API, programmable fax API, and wireless services.
+
+## Publisher: ********

```

Comment:
@fabianp5060 the publisher's name should follow the `Publisher: Publisher's Name` pattern.